specutils/tests/test_analysis.py

```diff
@@ -996,16 +997,16 @@ def test_moment_cube():
     noise = np.random.normal(0., 0.01, frequencies.shape) * u.Jy
     flux = g(frequencies) + noise
 
-    # use identical arrays in each spaxel. The purpose here is not to
+    # do not use identical arrays in each spaxel. The purpose here is not to
     # check accuracy (already tested elsewhere), but dimensionality.
 
-    flux_multid = np.broadcast_to(flux, [10,10,flux.shape[0]]) * u.Jy
+    flux_multid = np.broadcast_to(flux, [9, 10, flux.shape[0]]) * u.Jy
```

Comment:
This is the modified test case.

Reply:
I don't understand this particular line.  Why not just start with a shape of ``(9,10)`` instead of doing this broadcasting?

Reply:
You are using numpy random generator with seed above. I am afraid I will change more than I intended to.